assets/javascript/firebase.js

```diff
@@ -209,6 +229,22 @@ var firebaseBot = (function() {
     }
   }  
 
+  function updateLowGravityInfo(gameInfo) {
+    if (gameInfo.lowgravity) {
+      database.ref('games/' + gameInfo.gameId + "/playerOne/lowgravity").update({
```

Comment:
Same as the comment on line 242: This URL reference to the database can end at "/playerOne". It doesn't need the "/lowgravity" part. It will set the lowgravity property of playerOne to true, instead of adding another object with its own property of lowgravity.